templates/lobby.html

```diff
@@ -619,10 +619,11 @@
     drawingBoard.ontouchcancel = onTouchEnd;
 
     // Mouse input
-    drawingBoard.onmousedown = function (e) {
+    window.onmousedown = function (e) {
```

Comment:
Shouldn't this still be `drawingBoard.`? You don't wanna start drawing when clicking the chat, right?

Reply:
I suppose. Might have slipped in because I fixed the artefacts at the canvas edges from the drawing algorithm. Changing it now.